next-env.d.ts

```diff
@@ -1,6 +1,5 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
-/// <reference types="next/navigation-types/compat/navigation" />
```

Comment:
Can you run `npm install` ? What is your nodejs version ?

Reply:
Okay, (node V.18.12)